test/JDBC/expected/AUTO_ANALYZE-vu-prepare.out

```diff
@@ -0,0 +1,22 @@
+-- function to list all the babelfish catalogs that has not been analyzed manually
+CREATE FUNCTION auto_analyze_vu_prepare_catalogs_have_not_been_analyzed()
+RETURNS TABLE 
+AS RETURN
+(SELECT relname FROM pg_stat_all_tables WHERE schemaname = 'sys' and last_analyze IS NULL)
+GO
+
+-- function to list all the babelfish catalogs that has been analyzed manually
+CREATE FUNCTION auto_analyze_vu_prepare_catalogs_have_been_analyzed()
+RETURNS TABLE 
+AS RETURN
+(SELECT relname FROM pg_stat_all_tables WHERE schemaname = 'sys' and last_analyze IS NOT NULL)
+GO
```

Comment:
I dont understand why we need these functions

Reply:
removed the function in next commit.

Reply:
not a real use case from customer perspective but these were helping keep sql_validation_framework/expected_create.out clean

Reply:
@surendravishnoi those function were not invoking the analyze procedure introduced by us as those procedure is supposed to be run during create/upgrade script only by `sysadmin`.